pkg/injectors.go

```diff
@@ -0,0 +1,46 @@
+// +build wireinject
+
+package pkg
+
+import (
+	"github.com/google/wire"
+	"github.com/gorilla/mux"
+	"github.com/naftulikay/golang-webapp/pkg/dao"
+	"github.com/naftulikay/golang-webapp/pkg/database"
+	"github.com/naftulikay/golang-webapp/pkg/interfaces"
+	"github.com/naftulikay/golang-webapp/pkg/service"
+)
+
+func initializeApp(
+	router *mux.Router,
+	appCfg interfaces.AppConfig,
```

Comment:
Need to be able to bind `interfaces.ListenConfig` and `interfaces.MySQLConfig` out of `appCfg` here.